qa.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.6.0"
   s.add_dependency "activerecord-import"
   s.add_dependency "deprecation"
+  s.add_dependency "linkeddata"
```

Comment:
@jcoyne What do you suggest adding it as a development dependency and then making the code check to see if it is defined before requiring it? Or removing the tests and just leave the code untested? Or close the PR and put the code in ScholarSphere?